sys/include/net/gnrc/netif.h

```diff
@@ -41,6 +41,11 @@ extern "C" {
 #endif
 
 /**
+ * @brief   Identifier of the loopback interface
+ */
+#define GNRC_NETIF_LOOPBACK_PID     (-1)
```

Comment:
assertion fails at [`gnrc_ipv6.c:817`](https://github.com/OlegHahm/RIOT/blob/b7896804fb5887275ae6bfa1a33e8e03aafcb668/sys/net/gnrc/network_layer/ipv6/gnrc_ipv6.c#L817) when pinging `::1` or one of the node's unicast link local addresses in the `gnrc_networking` application.